pythonforandroid/recipe.py

```diff
@@ -151,6 +151,8 @@ def report_hook(index, blksize, size):
             attempts = 0
             while True:
                 try:
+                    import ssl
+                    ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
I don't think we want to skip certificate verification by default. Also usually packages are served through valid cert. When it's not the case it's very likely that you're missing some default root cert in your setup

Reply:
So he has to provide his instead of using the system cert?

Reply:
Well this is not providing any certificate to me, but says the client to not verify the server one which is dangerous

Reply:
It is. But a way would be something to do like this:
https://github.com/sfackler/rust-native-tls/pull/96/commits/1bbea92db6fa081ab0734f15e43696edf45dace1

Reply:
recipe.py doesn't run on android, it runs in the build system, when the distribution is being built, not checking the cert would open the door to build-time code injection, that would be hard to detect because it could happen in the code of any dependency of the app, imagine a wallet app being unable to trust any of its dependency has been downloaded without a MITM attach anymore? Publishing such a version of your app becomes way more stressful.